tasks/computer-vision/image-classification/MLproject

```diff
@@ -0,0 +1,14 @@
+name: ImageClassifier
+conda_env: conda.yaml
+
+entry_points:
+  # train Keras DL model
+  main:
+    parameters:
+
+command: "python ImageClassifier.py"
```

Comment:
@varsha6319 Did you mean to nest `command` under `main`? If so, please add four spaces before `command`.